filesystem/pom.xml

```diff
@@ -0,0 +1,129 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.frankframework</groupId>
+		<artifactId>frankframework-parent</artifactId>
+		<version>${revision}</version>
+	</parent>
+
+	<artifactId>frankframework-filesystem</artifactId>
+	<name>Frank!Framework FileSystems</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.frankframework</groupId>
+			<artifactId>frankframework-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>javax.servlet-api</artifactId>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Heeft hij deze nodig!?

Reply:
Ja, die had ik eerst ook verwijderd maar `FileHandler` haalt een HttpServletResponse uit de session, op regel 205, als deze conditie waar is: `("stream".equals(outputType) && isStreamResultToServlet())` (ik vermoed dat dit nog een stukje streaming is waar je in de toekomst vanaf wil, maar dit is iig wat het nu is)